src/components/corporate-footer/script.ts

```diff
@@ -17,15 +17,33 @@ Polymer({
     navItems: {
       type: Boolean,
       value: false
+    },
+    items: {
+      type: Array,
+      observer: 'changedItems'
     }
   },
   created: function() {
+    /* This condition was added based on solution in corporate-header */
+    if (window['params'].preload === 'false') {
+      var items = this.querySelectorAll('nav-item');
+      for(var i=0; i<items.length; i++) {
+        var item = items[i];
+        if ( !(item.children.length && item.children[0].nodeName === 'A') ) {
+          item.innerHTML = '<a href="' + item.getAttribute('location') + '">' + item.innerHTML + '</a>';
```

Comment:
Should we be able to add custom attributes here as well?
![image](https://user-images.githubusercontent.com/1199101/53479293-ff95ba80-3a78-11e9-9175-1c218d698510.png)

Reply:
Current solution applies the attributes on the nav-item as can be seen with target="_blank".
Seems to be an issue with the class thought.
I think we should merge this and create a issue with investigate class not being set.
As well as a task for setting the attributes on the a element maybe?